### DISCUSSION TITLE
Student proof

### DISCUSSION THREAD

**Discussion by ghost** [community/community#120529] [GitHub Education]
### Hi everyone,

How can I prove that I am a student when I can only upload one photo, and my school name and the valid date are on different sides of my student id card?

**Comment by gurjeetsinghvirdee** [comment]
Hey @yuno-programing 
If possible, try taking a clear photo of both sides of your student ID card and merge them into a single image. Ensure that both the school name and valid date are clearly visible.

**Comment by samus-aran** [comment]
Hi @yuno-programing 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [I have a double-sided Student ID with the date on the other side. How do I submit this as evidence?](https://github.com/orgs/community/discussions/111352#double-sided-student-id-with-the-date-on-the-other-side)

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍
